bitnami/common/templates/_capabilities.tpl

```diff
@@ -15,6 +15,17 @@ Return the target Kubernetes version
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return the appropriate apiVersion for policy.
+*/}}
+{{- define "common.capabilities.policy.apiVersion" -}}
+{{- if .Capabilities.APIVersions.Has "policy/v1" }}
+{{- print "policy/v1" -}}
+{{- else -}}
+{{- print "policy/v1beta1" -}}
+{{- end -}}
```

Comment:
According to the [Deprecation guide](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#poddisruptionbudget-v125), manifests should be migrated from `policy/v1beta1` to `policy/v1` starting with K8s `v1.21`.
Should we use this instead?
```suggestion
{{- if semverCompare "<1.21-0" (include "common.capabilities.kubeVersion" .) -}}
{{- print "policy/v1beta1" -}}
{{- else -}}
{{- print "policy/v1" -}}{{- end -}}
```
Just to follow the same approach used for the rest of capability functions.
cc @javsalgar